src/commands/ft_create_parser.cc

```diff
@@ -446,14 +555,137 @@ absl::StatusOr<data_model::IndexSchema> ParseFTCreateArgs(
   if (res) {
     return absl::InvalidArgumentError(NotSupportedParamErrorMsg(kFilterParam));
   }
-  VMSDK_RETURN_IF_ERROR(ParseLanguage(itr, index_schema_proto));
-  VMSDK_RETURN_IF_ERROR(ParseScore(itr, index_schema_proto));
-  VMSDK_ASSIGN_OR_RETURN(
-      res, vmsdk::IsParamKeyMatch(kPayloadFieldParam, false, itr));
-  if (res) {
-    return absl::InvalidArgumentError(
-        NotSupportedParamErrorMsg(kPayloadFieldParam));
+  // Parse global text parameters before SCHEMA
```

Comment:
Did we test out FT.CREATE in an integration test as we talked about?

Example: https://github.com/valkey-io/valkey-search/blob/main/integration/test_query_parser.py